helm_resource/helm-apply-helper.py

```diff
@@ -8,6 +8,7 @@
 import subprocess
 import sys
 from typing import Dict
+from namespacing import add_default_namespace
```

Comment:
Genuinely shocked this worked without any `PYTHONPATH` shenanigans 😂

Reply:
tbh i don't really understand python import semantics